jellyfin/jellyfin-androidtv
# Add Mockk and one test to exercise the new dep

**Changes**
    Add Mockk to test framework deps
    
    Add Mockk as this is aware of Kotlin inheritence unlike Mockito.
    This primarily shows up where classes are not marked open. We can
    either:
    - Mark all unit tested classes open (least viable option)
    - Add boilerplate to Mockito each time we test a Kotlin class
    - Add a mocking framework that is Kotlin aware.
    
    In this case I've opted for the latter, since it also avoids us having
    to look up the boilerplate for static mocks...etc.

I've also added a single set of simple tests to exercise Mockk / demo it's working. This blurs single change per PR, but without it it's not really possible to prove Mockk is working.

For more details see each commit message

**To Test**
- Run DeviceUtils test
